### PR TITLE
Add prescriptions documentation

### PR DESCRIPTION
**Pull request by fabyan-alexander**
### Overview
beta_prescriptions_overview

### Endpoints
beta_prescriptions_endpoints

### CODE REVIEW

**Comment by fabyan-alexander** [comment]
[[docs] Add Prescription endpoints](https://trello.com/c/lT7t3dK0/217-docs-add-prescription-endpoints)

**Review by Vinniekun — APPROVED** [review]

**Review by palomacione — APPROVED** [review]

File: docs/beta_prescriptions_endpoints.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+      'https://api.withleaf.io/services/beta/prescriptions/api/users/{leafUserId}/climateFieldView'
+  ```
+
+  </TabItem>
+</Tabs>
+
+
+#### Response
+
+The response is json with the id of the file uploaded to the provider .
```
Comment (ricardosantacattarina) [inline docs/beta_prescriptions_endpoints.md:257]:
```suggestion
The response is json with the id of the file uploaded to the provider.
```

File: docs/beta_features_overview.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
 The beta features contains experimental features that we are working in progress but can be used by our users.
 The services available in this section can be updated as we improve 
 
 Currently, we are working in the following features:
 
+[1]: #machines
+
 ## Machines
 
 This feature will fetch machine data from the providers like (JohnDeere, ClimateFieldView, CNHI) and store this data.
```
Comment (ricardosantacattarina) [inline docs/beta_features_overview.md:15]:
```suggestion
This feature will fetch machine data from the providers (like John Deere, Climate FieldView, CNHI) and store this data.
```

**Review by ricardosantacattarina — APPROVED** [review]
Nice docs! I only noticed things from previous PRs, sorry :sweat_smile:

File: docs/beta_features_overview.md (hunk 2 above)
Comment (fabyan-alexander) [reply docs/beta_features_overview.md]: Thanks! Fixed in https://github.com/Leaf-Agriculture/docs/pull/171/commits/7e1d98bc3779174a86ab3460cb79a884fd8276c3

File: docs/beta_features_overview.md [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
-[1]: #machines
-
 
 The beta features contains experimental features that we are working in progress but can be used by our users.
 The services available in this section can be updated as we improve 
 
 Currently, we are working in the following features:
 
+[1]: #machines
+
```
[the remark this answers is not in the document]
Comment (fabyan-alexander) [reply docs/beta_features_overview.md:12]: Yeah! My bad and thanks for pointing that!
I'll remove that in this commit and we will review it again when we add the machine operations :rocket: 
Removed in https://github.com/Leaf-Agriculture/docs/pull/171/commits/7e1d98bc3779174a86ab3460cb79a884fd8276c3

**Pull request merged**